__tests__/app.test.js

```diff
@@ -81,6 +81,43 @@ describe("Topics Endpoint", () => {
 });
 
 describe("Articles Endpoint", () => {
+  describe("GET:/api/articles", () => {
+    test("GET:200 - Responds with an array containing correctly formated article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+
+          expect(articles.length).not.toBe(0);
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number),
+              })
+            );
+          });
+        });
+    });
+    test("GET:200 - Responds with an array containing correctly formated article objects sorted by date in descending order", () => {
```

Comment:
What do you mean by correctly formatted here? The only thing you check is whether the articles are sorted by date.